### ISSUE TITLE
Add Support for Micro Compilation and Testing

### ISSUE THREAD

**Issue by javmorcas** [rticommunity/rtiperftest#78]
This issue depends on #37 and #36

**TODO**:
- [ ] Support for C++ Traditional
- [ ] First CR [Code Review]()
- [ ] Automatic install testing
- [ ] Documentation changes
- [ ] Examples
- [ ] Full [Code Review]()
- [ ] Blogpost

**Comment by javmorcas** [comment]
Merged into master #179
